.github/workflows/containers/github-action-ci-tooling/Dockerfile

```diff
@@ -0,0 +1,58 @@
+ARG LLVM_VERSION=21.1.0
+
+FROM docker.io/library/ubuntu:24.04 AS llvm-downloader
+ARG LLVM_VERSION
+
+RUN apt-get update && \
+    apt-get install -y wget xz-utils && \
+    wget --progress=bar:force -O llvm.tar.xz https://github.com/llvm/llvm-project/releases/download/llvmorg-${LLVM_VERSION}/LLVM-${LLVM_VERSION}-Linux-X64.tar.xz && \
+    mkdir -p /llvm-extract && \
+    tar -xvJf llvm.tar.xz -C /llvm-extract LLVM-${LLVM_VERSION}-Linux-X64/bin/ && \
+    rm llvm.tar.xz
+
+
+FROM docker.io/library/ubuntu:24.04 AS base
+ENV LLVM_SYSROOT=/opt/llvm
+
+# Need nodejs for some of the GitHub actions.
+# Need git for git-clang-format.
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    git \
+    nodejs \
+    sudo \
+    # These are needed by the premerge pipeline.
+    # Pip is used to install dependent python packages.
+    python3-pip \
+    python-is-python3 && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /home/gha
```

Comment:
I don't think this work directory is valid now that we aren't creating the user? Keeping the default (ie just deleting this line) should be fine.